app/controllers/forms_controller.rb

```diff
@@ -43,6 +43,19 @@ def update
     end
   end
 
+  def new
+    form = Form.create! name: "new-form"
```

Comment:
Hmmm. Is there a way to do this without the default name?

Name is a required attribute, but it also has to be unique.  It seems that someone leaving this name unchanged might cause issues.